crates/ruff/src/version.rs

```diff
@@ -68,6 +68,11 @@ pub(crate) fn version() -> VersionInfo {
     }
 }
 
+/// Used for testing the --version command
+pub fn version_string() -> String {
+    format!("{}", version())
+}
+
```

Comment:
I would prefer using `filters` in the test over making this function and the entire module public.

Reply:
Good point, I can probably just use a regex in the filter rather than `re.escap`ing the exact version in the filter... thanks :)